packages/ExpandCollapse/PanelWrapper/PanelWrapper.jsx

```diff
@@ -29,11 +25,53 @@ const parseHeader = text => {
       if (index % 2 === 0) {
         return line
       }
-      return <Text.Sup key={line}>{line}</Text.Sup>
+      return <sup key={line}>{line}</sup>
     })
   return t
 }
 
+const HeaderButtonClickable = styled.button(buttons.noStyle, ({ panelDisabled }) => ({
+  width: '100%',
+  textAlign: 'left',
+  ...(panelDisabled && { background: colorAthensGrey, cursor: 'default' }),
+}))
+
+const CaretContainer = styled.div(({ isDisabled }) => ({
+  ...(isDisabled && { visibility: 'hidden' }),
+}))
+
+const HeaderContainer = styled.div(({ direction }) => ({
```

Comment:
Shouldn't this be shared as Flexbox was shared before? I guess should Flexbox component be updated to styled-components, as well?